style-guide/src/components/design-nav/design-nav.js

```diff
@@ -2,9 +2,13 @@ import React from 'react';
 import { Link } from 'gatsby';
 import styles from '../../styles/index.module.scss';
 
-const NavLink = (props) => (
+const NavLink = props => (
   <div className={styles.navLink}>
-    <Link to={props.to} exact={props.exact} activeClassName={styles.linkActive}>
+    <Link
+      to={props.to}
+      partiallyActive={props.exact ? false : true}
```

Comment:
Using `exact` props to check for exact path. If we don't, refreshing the page will make the active link disappear.